pkg/infra/tracing/tracing.go

```diff
@@ -47,21 +53,26 @@ func (ts *TracingService) Init() error {
 
 func (ts *TracingService) parseSettings() {
 	var section, err = ts.Cfg.Raw.GetSection("tracing.jaeger")
-	if err != nil {
-		return
+	if err == nil {
+		ts.address = section.Key("address").MustString("")
+		ts.customTags = splitTagSettings(section.Key("always_included_tag").MustString(""))
+		ts.samplerType = section.Key("sampler_type").MustString("")
+		ts.samplerParam = section.Key("sampler_param").MustFloat64(1)
+		ts.zipkinPropagation = section.Key("zipkin_propagation").MustBool(false)
+		ts.disableSharedZipkinSpans = section.Key("disable_shared_zipkin_spans").MustBool(false)
+		ts.samplingServerURL = section.Key("sampling_server_url").MustString("")
 	}
 
-	ts.address = section.Key("address").MustString("")
+	if ts.address == "" {
+		host := os.Getenv(envJaegerAgentHost)
+		port := os.Getenv(envJaegerAgentPort)
```

Comment:
One question. What happens in the client if only one of these are supplied, is it still enabled with default host or port? Depending on the answer maybe only enable if address not empty or (host or port is not empty) and let the client do it's thing?

Reply:
Updated the "if" condition. Now if either host or port variable is present - it overrides the address from the INI file. 
If host is set - address would be host + default port.
If port is set  - address would be default host (localhost) + port
If both are set - address would be host + port
If none is set - address would be taken from the INI file
If it is empty - tracer would be disabled (default)

Reply:
This should work. I guess the only confusing with this code is that `ts.address` is never used if `JAEGER_AGENT_*` env variables are defined. But I guess we can live with that?

Reply:
No biggie really so resolving this.